src/mappings/oba-vt.sssom.tsv

```diff
@@ -2306,4 +2306,7 @@ VT:0003620	skos:exactMatch	OBA:0000173	HumanCurated
 VT:0001661	skos:exactMatch	OBA:0003653	HumanCurated
 VT:0010719	skos:exactMatch	OBA:2040162	HumanCurated
 VT:0010718	skos:exactMatch	OBA:2040163	HumanCurated
-VT:0010721	skos:exactMatch	OBA:2040164	HumanCurated
\ No newline at end of file
+VT:0010721	skos:exactMatch	OBA:2040164	HumanCurated
+VT:0015008	skos:exactMatch	OBA:0004528	HumanCurated
+VT:0010483	skos:broadMatch	OBA:0004147	HumanCurated
+VT:0007086	skos:exactMatch	OBA:0006021	HumanCurated
```

Comment:
Heart ventricle septum thickness and interventricular septum thickness refer to the same entity.